include/dxc/Support/HLSLOptions.td

```diff
@@ -451,6 +451,7 @@ def rw_extract_entry_uniforms : Flag<["-", "/"], "extract-entry-uniforms">, Grou
   HelpText<"Move uniform parameters from entry point to global scope">;
 def rw_remove_unused_globals : Flag<["-", "/"], "remove-unused-globals">, Group<hlslrewrite_Group>, Flags<[RewriteOption]>,
   HelpText<"Remove unused static globals and functions">;
```

Comment:
Should probably remove the word `static` here, since this option applies to all globals, not just static ones.  We could have one that only applies to static globals/functions (doesn't change cbuffer layout, for instance) in the future, and perhaps call it `-remove-unused-static-globals`.

Reply:
Or perhaps this option should be folded into the -remove-unused-functions, since the purpose of that one is not to change cbuffer layout, but remove the extra code that doesn't contribute to the compile of an entry point.

Reply:
Oh, I see -remove-unused-functions removes static globals as well, so nevermind!